dd-sdk-android/src/test/kotlin/com/datadog/android/core/internal/lifecycle/ProcessLifecycleCallbackTest.kt

```diff
@@ -72,6 +76,14 @@ internal class ProcessLifecycleCallbackTest {
                 )
             )
 
+        whenever(
```

Comment:
this (and in other tests) is needed because `enqueueUniqueWork` and `cancelAllWorkByTag` have `@NonNull` return value as part of the contract and check of the contract is enforced by the code produced by the 1.4.x compiler.